docs/apps/components/data-grid.md

```diff
@@ -0,0 +1,208 @@
+3. Click on `Save`.
+4. On the main screen click on `Save` again.
+5. Click on `Preview` to see the code in action.
+<img src= "/apps/components/img/check1.png">
+
+!!! Example
+    1. `this.addClassName(fields, className)`
+          * Go to Form Attribute :material-menu-right: Style Sheets :material-menu-right: add the class
+            ```
+            .abc{ // abc is the class name
+            background-color: red;
+            }
+            ```
+          * Follow the steps mentioned above, under Steps to use the methods for the Page Builder components
+          * ```
+            this.addClassName('aggrid_nd93m43c', 'abc')
+            ```
+
+    2. `this.removeClassName(fields, className)`
+        ```js
+        this.removeClassName('aggrid_nd93m43c', 'abc')
+        ```
+    
+    3. `this.getValue(fieldName)`
+        ```js
+        var dataname = this.getValue('aggrid_nd93m43c');
+        console.log('getValue', dataname);
+        ```
+    
+    4. `this.getValues()`
+        ```js
+        var data = this.getValues();
+        console.log(data);
+        ```
+    
+    5. `this.hide(fields)`
+        ```js
+        var fields= ['aggrid_nd93m43c']
+        this.hide(fields)
+        ```
+    
+    6. `this.show(fields)`
+        ```js
+        var fields= ['aggrid_nd93m43c']
+        this.show(fields)
+        ```
+    
+    7. `this.disable(['fields'])`
+        ```js
+        this.disable(['aggrid_nd93m43c'])
+        ```
+    
+    8.  `this.getValue('fieldName')`
+        ```js
+        var dataname = this.getValue('aggrid_nd93m43c');
+        console.log('getValue', datanamee);
+        ```
+    9. `this.getComponent('component_ID')`
+        ```js
+        var dataname = this.getComponent('aggrid_nd93m43c');
+        console.log('getComponent', dataname);
+        ```
+
+## Config
+
+| **Name**|**Description**|**Icon**|
+|---------|---------------|--------|
+|**ID**| The data-grid component's unique identifier|<img src= "/apps/components/img/input_id.png">|
+|**Width**| [**Optional**] Width of the field|<img src= "/apps/components/img/input_width.png">|
+|**Status Bar**|Displays information about the number of rows and columns|<img src= "/apps/components/img/datagrid_statusbar.png">|
+|**Side Bar**|Displays the side bar with Columns and Filters|<img src= "/apps/components/img/datagrid_sidebar.png">|
+|**Enable Range Selection**|Enable to select multiple cells in the data-grid and it provides you with minimum value, maximum value, average, sum, count|<img src= "/apps/components/img/datagrid_rangeselection.png">|
+|**Suppress Context Menu**|When disabled, it enables the mouse right-click which has options like export into `.CSV` file or `Excel`, copy or copy with headers etc|<img src= "/apps/components/img/datagrid_suppress.png">|
+|**Context Menu**|You can select from various options to add to your context menu|<img src= "/apps/components/img/datagrid_contextmenu.png">|
```

Comment:
We need to specify here that this option is ONLY effective if suppress Context Menu is Disabled.

Reply:
done